### PR TITLE
Test/initial actions reducers test

### LINKED ISSUE #60: Create rudimentary tests for actions and reducers
- this is to make sure dependeicnies are installed correctly
- this adds a test to an existing action
- this adds a test to an existing reducer
- this does NOT test new functionality yet

### PR DESCRIPTION
**Pull request by mainlyetcetera**
## *Purpose:*

- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*

- download and install [jest](https://jestjs.io/docs/getting-started) as a dependency
  - this is to unit-test on the front-end
- add rudimentary test of the `setup-user` action
- add rudimentary tests of the `rootReducer` reducer

## *What are the relevant tickets?*
 
Closes #60

## *Bugs:*

- `database.json` was deleted again, watch in future

## Checklist:
- [x] I have performed a self-review of my own code
- [x] I have used tests to prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my change
- [x] I used a rebase workflow

## *Notes:*
- no further notes

### CODE REVIEW

**Review by LiamCusack — APPROVED** [review]
looks good to me!

File: code/web/src/setup/store.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { createStore, applyMiddleware } from 'redux'
 import thunk from 'redux-thunk'
 
 // App Imports
+// here is where all the reducers live
```
Comment (lbmerchant93) [inline code/web/src/setup/store.js:7]: Thanks for creating this note! Makes finding these easier!

File: code/web/src/setup/reducer.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  it('should empty state when resetting', () => {
+    const passedState = {}
+
+    const actual = rootReducer(passedState, 'RESET')
+
+    expect(actual.crates.list).toEqual([])
+    expect(actual.subscription.item).toEqual({})
+  })
+})
+
```
Comment (lbmerchant93) [inline code/web/src/setup/reducer.test.js:26]: Just a note for all of us not to forget to test for a default "return state if no action is given"

File: code/web/src/modules/user/api/actions.test.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    const actual = actions.setUser('SET_USER', user)
+
+    expect(actual).toEqual({
+      type: 'AUTH/SET_USER',
+      user
+    })
+  })
+})
+
+
```
Comment (lbmerchant93) [inline code/web/src/modules/user/api/actions.test.js:20]: Great start on testing! Seems like you understand the setting up of testing! I feel like we should be able to knock out the rest of the testing fairly quickly

**Review by lbmerchant93 — APPROVED** [review]
Looks great Eric! I think this is a really great start!

**Review by christina-cook — APPROVED** [review]
Nice start on the testing! Looks like everything is set up correctly.

**Pull request merged**